src/shared/tpm2-util.c

```diff
@@ -596,8 +598,65 @@ static int tpm2_get_best_pcr_bank(
         return 0;
 }
 
+static int tpm2_make_hmac_session(
+                ESYS_CONTEXT *c,
+                ESYS_TR tpmKey,
+                ESYS_TR *ret_session) {
+
+        static const TPMT_SYM_DEF symmetric = {
+                .algorithm = TPM2_ALG_AES,
+                .keyBits = {
+                        .aes = 128
+                },
+                .mode = {
+                        .aes = TPM2_ALG_CFB,
+                }
```

Comment:
doesn't matter at all, but we usually place a trailing `,` into all struct inits, just to be safe for the future.

Reply:
We already fill in the struct completely, though.

Reply:
the reader won't know that though.

but doesn't matter at all. (you could earn bonus points though if you also add the `,` to the `keyBits` struct init a few lines above)

Reply:
I added commas for consistency after all.